packages/browser/src/integrations/graphqlClient.ts

```diff
@@ -0,0 +1,48 @@
+import { SEMANTIC_ATTRIBUTE_SENTRY_OP, defineIntegration, spanToJSON } from '@sentry/core';
+import type { IntegrationFn } from '@sentry/types';
+import { parseGraphQLQuery } from '@sentry/utils';
+
+interface GraphQLClientOptions {
+  endpoints: Array<string>;
+}
+
+const INTEGRATION_NAME = 'GraphQLClient';
+
+const _graphqlClientIntegration = ((options: GraphQLClientOptions) => {
+  return {
+    name: INTEGRATION_NAME,
+    setup(client) {
+      client.on('spanStart', span => {
+        const spanJSON = spanToJSON(span);
+
+        const spanAttributes = spanJSON.data || {};
+
+        const spanOp = spanAttributes[SEMANTIC_ATTRIBUTE_SENTRY_OP];
+        const isHttpClientSpan = spanOp === 'http.client';
+
+        if (isHttpClientSpan) {
+          const httpUrl = spanAttributes['http.url'];
```

Comment:
We should use this heuristic here (this is pretty annoying, but...):

```suggestion
          const httpUrl = spanAttributes[SEMANTIC_ATTRIBUTE_URL_FULL] || spanAttributes['http.url'];
```

`SEMANTIC_ATTRIBUTE_URL_FULL` is exported from core!